Resync `html/syntax` from WPT Upstream

Ahmad-S792 opened the pull request:
#### 72bbdfac7a2e6bbb1f6ed7dbc8326846720792bc
<pre>
Resync `html/syntax` from WPT Upstream
<a href="https://bugs.webkit.org/show_bug.cgi?id=299264">https://bugs.webkit.org/show_bug.cgi?id=299264</a>
<a href="https://rdar.apple.com/161066021">rdar://161066021</a>

Reviewed by Anne van Kesteren.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/1d3fbb842fe3ad007c5f920f717455c83ce557ed">https://github.com/web-platform-tests/wpt/commit/1d3fbb842fe3ad007c5f920f717455c83ce557ed</a>

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/inheritance-bogus-meta-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/inheritance-bogus-meta-utf-8-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/inheritance-bogus-meta-utf-8.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/inheritance-bogus-meta.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-charset-double-quotes-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-charset-double-quotes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-charset-no-quotes-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-charset-no-quotes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-charset-single-quotes-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-charset-single-quotes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-commented-out-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-commented-out.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-http-equiv-attribute-order-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-http-equiv-attribute-order.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-http-equiv-double-quotes-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-http-equiv-double-quotes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-http-equiv-no-quotes-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-http-equiv-no-quotes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-http-equiv-single-quotes-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-http-equiv-single-quotes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/resources/bogus-charset-http-valid-meta.py: Added.
(main):
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/resources/bogus-charset-http.py: Added.
(main):
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/resources/bogus-charset.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/resources/w3c-import.log: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/syntax/writing/elements-0/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/html-element-list.js: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing-html-fragments/the-input-byte-stream-015-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing-html-fragments/the-input-byte-stream-015.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing-html-fragments/the-input-byte-stream-015.html.headers: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/ambiguous-ampersand-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/ambiguous-ampersand.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_innerHTML_tests_innerHTML_1-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_innerHTML_tests_innerHTML_1.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_innerHTML_webkit02-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_innerHTML_webkit02.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_menuitem-element.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_menuitem-element_run_type=uri-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_menuitem-element_run_type=write-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_menuitem-element_run_type=write_single-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests1.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests10.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests10_run_type=uri-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests10_run_type=write-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests10_run_type=write_single-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests18.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests18_run_type=uri-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests18_run_type=write-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests18_run_type=write_single-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests1_run_type=uri-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests1_run_type=write-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests1_run_type=write_single-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests7.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests7_run_type=uri-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests7_run_type=write-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests7_run_type=write_single-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests9.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests9_run_type=uri-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests9_run_type=write-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests9_run_type=write_single-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_webkit02.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_webkit02_run_type=uri-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_webkit02_run_type=write-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_webkit02_run_type=write_single-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/misnested-form-in-template-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/misnested-form-in-template.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/template/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/template/w3c-import.log: Copied from LayoutTests/imported/w3c/web-platform-tests/html/syntax/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/zero-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/zero.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/expect-no-linked-resources/no-speculative-fetch.tentative.optional-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/expect-no-linked-resources/no-speculative-fetch.tentative.optional.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/expect-no-linked-resources/resources/no-speculative-fetch.sub.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/expect-no-linked-resources/resources/no-speculative-fetch.sub.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/expect-no-linked-resources/resources/stash.py: Added.
(main):
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/expect-no-linked-resources/resources/w3c-import.log: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/syntax/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/expect-no-linked-resources/w3c-import.log: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/syntax/writing/text/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/generated/document-write/nested-template-shadowrootmode-2.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/generated/document-write/script-src-referrerpolicy-no-referrer.tentative.sub-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/generated/document-write/template-img-src.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/tools/generate.py:
(generate_tests):
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/writing/elements-0/contains.json: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/writing/text/contains.json: Removed.
* LayoutTests/tests-options.json:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/html/syntax/parsing-html-fragments/the-input-byte-stream-003-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/html/syntax/parsing-html-fragments/the-input-byte-stream-004-expected.txt:

Canonical link: <a href="https://commits.webkit.org/300336@main">https://commits.webkit.org/300336@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/d127c53a1437833ab365533ec85cfbf93888bc48

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/122172 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/41875 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/32544 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/128745 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/74266 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/d9b6f049-394b-405b-9701-4048f6bb2062) 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/124048 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/42589 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/50468 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/92858 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/61733 "Passed tests") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/5a9b50db-0590-47d5-9103-1cef7d5da305) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/125124 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/33970 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/109399 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/73514 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/8be0020d-8cb5-4aff-bcd4-d8c1bb3de34e) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/32982 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/27564 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/72233 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/103478 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/27755 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/131496 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/49111 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/37361 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/101427 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/49485 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/105610 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/101296 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/25682 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/46667 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/24780 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/45862 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/48968 "Built successfully") | | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/48438 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/51788 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/50118 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

annevk reviewed (approved):
Looks great, modulo some Linux expectations. Thanks for tackling this!

Ahmad-S792 committed:
Resync `html/syntax` from WPT Upstream
https://bugs.webkit.org/show_bug.cgi?id=299264
rdar://161066021

Reviewed by Anne van Kesteren.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/1d3fbb842fe3ad007c5f920f717455c83ce557ed

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/inheritance-bogus-meta-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/inheritance-bogus-meta-utf-8-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/inheritance-bogus-meta-utf-8.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/inheritance-bogus-meta.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-charset-double-quotes-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-charset-double-quotes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-charset-no-quotes-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-charset-no-quotes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-charset-single-quotes-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-charset-single-quotes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-commented-out-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-commented-out.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-http-equiv-attribute-order-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-http-equiv-attribute-order.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-http-equiv-double-quotes-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-http-equiv-double-quotes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-http-equiv-no-quotes-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-http-equiv-no-quotes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-http-equiv-single-quotes-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/meta-http-equiv-single-quotes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/resources/bogus-charset-http-valid-meta.py: Added.
(main):
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/resources/bogus-charset-http.py: Added.
(main):
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/resources/bogus-charset.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/resources/w3c-import.log: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/syntax/writing/elements-0/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/charset/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/html-element-list.js: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing-html-fragments/the-input-byte-stream-015-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing-html-fragments/the-input-byte-stream-015.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing-html-fragments/the-input-byte-stream-015.html.headers: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/ambiguous-ampersand-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/ambiguous-ampersand.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_innerHTML_tests_innerHTML_1-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_innerHTML_tests_innerHTML_1.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_innerHTML_webkit02-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_innerHTML_webkit02.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_menuitem-element.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_menuitem-element_run_type=uri-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_menuitem-element_run_type=write-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_menuitem-element_run_type=write_single-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests1.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests10.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests10_run_type=uri-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests10_run_type=write-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests10_run_type=write_single-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests18.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests18_run_type=uri-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests18_run_type=write-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests18_run_type=write_single-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests1_run_type=uri-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests1_run_type=write-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests1_run_type=write_single-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests7.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests7_run_type=uri-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests7_run_type=write-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests7_run_type=write_single-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests9.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests9_run_type=uri-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests9_run_type=write-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_tests9_run_type=write_single-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_webkit02.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_webkit02_run_type=uri-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_webkit02_run_type=write-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html5lib_webkit02_run_type=write_single-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/misnested-form-in-template-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/misnested-form-in-template.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/template/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/template/w3c-import.log: Copied from LayoutTests/imported/w3c/web-platform-tests/html/syntax/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/zero-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/zero.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/expect-no-linked-resources/no-speculative-fetch.tentative.optional-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/expect-no-linked-resources/no-speculative-fetch.tentative.optional.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/expect-no-linked-resources/resources/no-speculative-fetch.sub.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/expect-no-linked-resources/resources/no-speculative-fetch.sub.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/expect-no-linked-resources/resources/stash.py: Added.
(main):
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/expect-no-linked-resources/resources/w3c-import.log: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/syntax/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/expect-no-linked-resources/w3c-import.log: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/syntax/writing/text/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/generated/document-write/nested-template-shadowrootmode-2.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/generated/document-write/script-src-referrerpolicy-no-referrer.tentative.sub-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/generated/document-write/template-img-src.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/speculative-parsing/tools/generate.py:
(generate_tests):
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/writing/elements-0/contains.json: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/writing/text/contains.json: Removed.
* LayoutTests/tests-options.json:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/html/syntax/parsing-html-fragments/the-input-byte-stream-003-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/html/syntax/parsing-html-fragments/the-input-byte-stream-004-expected.txt:

Canonical link: https://commits.webkit.org/300336@main
